flake.nix

```diff
@@ -124,6 +124,26 @@
                     value = addWerror v;
                   })
                   x.components."${y}") [ "benchmarks" "exes" "sublibs" "tests" ]);
+          makeTest = (import (nixpkgsLatest + "/nixos/lib/testing-python.nix") { inherit system; }).makeTest;
+          makeHydraVMTest = { name, systemPackages, ... }: makeTest {
+            inherit name;
+            nodes = {
+              testNode = { ... }: {
+                environment = { inherit systemPackages; };
+                networking.firewall.enable = false;
+                virtualisation = {
+                  cores = 2;
+                  memorySize = 8192;
```

Comment:
Does this now always only run with 4 cores and 8GB RAM even if we have more available?

Reply:
I assume so

Reply:
There are examples in nixpkgs where they do not specify the core count or the memory. What does it do then?

https://github.com/NixOS/nixpkgs/blob/27959498087a941c298055a059511f5843de5505/nixos/tests/tigervnc.nix#L7